template/src/main/scala/ch.epfl.scala.index.views/package.scala

```diff
@@ -108,7 +108,8 @@ package object html {
     val prefix = "scala_"
     project.targets
       .collect { case t if t.startsWith(prefix) => t.drop(prefix.length) }
-      .to[List].sorted(Ordering.String.reverse)
+      .map(SemanticVersion.apply).flatten
```

Comment:
It looks like it collides with `SemanticVersion.apply` defined for its case class:

`[error] /home/adam/IdeaProjects/scaladex/template/src/main/scala/ch.epfl.scala.index.views/package.scala:111: type mismatch;
[error]  found   : (major: Long, minor: Long, patch: Option[Long], patch2: Option[Long], preRelease: Option[ch.epfl.scala.index.model.PreRelease], metadata: Option[String])ch.epfl.scala.index.model.SemanticVersion <and> (version: String)Option[ch.epfl.scala.index.model.SemanticVersion]
[error]  required: String => scala.collection.GenTraversableOnce[?]
[error]       .flatMap(SemanticVersion.apply)`